dev-docs/workflows/upgrade-kubernetes.md

```diff
@@ -27,6 +27,8 @@ curl -qL https://mcr.microsoft.com/v2/oss/kubernetes/azure-cloud-node-manager/ta
 
 Normally renovate will handle the upgrading of Kubernetes dependencies.
 
+Also, don't forget to update the Kubernetes versions tested in the CI. (e.g. release, weekly E2E tests)
```

Comment:
I vaguely remember @daniel-weisse adding a script somewhere?

Reply:
I added one for Go versions, not sure about Kubernetes.
Maybe we can add a similar command here tho